.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Can you remove this file and add `.vscode` to the `.gitignore`?

Reply:
you got it.